src/components/HierarchicalFacetContent.tsx

```diff
@@ -0,0 +1,68 @@
+import { DisplayableFacet } from '@yext/search-headless-react';
+import { HierarchicalFacetProps } from './FacetProps';
+import {
+  builtInCollapsibleLabelCssClasses,
+  CollapsibleLabel, CollapsibleLabelCssClasses,
+  CollapsibleSection,
+  FilterGroupProvider,
+  HierarchicalFacetDisplay
+} from './Filters';
+import { useMemo } from 'react';
+import { twMerge } from '../hooks/useComposedCssClasses';
+
+/**
+ * A component that displays the content of a hierarchical facet.
+ *
+ * @param props - props to render the component
+ * @returns A React component for the content of a standard facet
```

Comment:
this says 'standard' facet when it should say 'hierarchical'

Reply:
done